sys/tiny_strerror/tiny_strerror.c

```diff
@@ -234,18 +233,19 @@ const char *tiny_strerror(int errnum)
         errnum = -errnum;
     }
 
-    if (((unsigned)errnum < ARRAY_SIZE(lookup))
-            && (lookup[(unsigned)errnum] != NULL)) {
-        retval = lookup[(unsigned)errnum];
+    if (((unsigned)errnum >= ARRAY_SIZE(lookup))
+            || (lookup[(unsigned)errnum] == NULL)) {
+        const char *errstr = "-unknown";
+        return errstr + offset;
```

Comment:
```suggestion
        return "unknown";
```

Reply:
OK, I dropped the differentiation between positive and negative numbers for unknown codes.